jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-3.yaml

```diff
@@ -155,18 +155,18 @@
           ret=0
 
           ssh root@$admin "
-          export cloud=$cloud ;
-          export hw_number=$hw_number ;
-          export UPDATEREPOS=$UPDATEREPOS ;
-          export UPDATEBEFOREINSTALL=$UPDATEBEFOREINSTALL ;
-          export TESTHEAD=$TESTHEAD ;
-          export cloudsource=$cloudsource ;
-          export nodenumber=$nodenumber ;
-          export hacloud=$hacloud ;
-          export clusterconfig=$clusterconfig ;
-          export shared_storage_ip=$shared_storage_ip ;
-          export want_node_aliases=controller=3,compute=3;
-          export scenario=\"/root/scenario.yml\" ;
+          export cloud=$cloud
```

Comment:
such cleanup should be a separate commit. you can use `git reset --patch HEAD^` and `git commit --amend` to split things out.